GeneticInheritanceGraphLibrary/tables.py

```diff
@@ -674,6 +678,102 @@ def random_match_pos(self, rng):
                             return self.MRCApos(u[0] - loc - 1, v[0] - loc - 1, False)
                 loc -= x[1] - x[0]
 
+    def plot(
+        self,
+        loc=None,
+        figsize=(10, 5),
```

Comment:
Instead of specifying fig size in the params, I think it would be useful to have `ax=None`. If None, you call `plt.subplots()` with the Matplotlib default figsize. If anyone want to change the fig size, they call `plt.subplots(figsize=my_bespoke_figsize)` themselves and pass `ax` in to the function.

This helps because then a user can pass their own `ax` in case they want to put the plot in a subfigure, or pass it to a pdf, or whatever. It also reduces the need for extra graphics parameters.

Reply:
Also any parameters after `loc` (or maybe `ax` can be keyword-only, I think (place an asterisk in the parameter list)